client/data/multi-currency/hooks.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { STORE_NAME } from '../constants';
+
+export const useAvailableCurrencies = () =>
+	useSelect( ( select ) => {
+		const { getAvailableCurrencies, isResolving } = select( STORE_NAME );
+
+		return {
+			deposit: getAvailableCurrencies(),
```

Comment:
This is probably something that is left from copy-paste, but what do you think of using a different name here instead of `deposit`? The same applies to `useEnabledCurrencies`.

Reply:
Good catch, I missed updating those after the copy/paste. Updated in https://github.com/Automattic/woocommerce-payments/pull/1856/commits/491b258ba28a0cde79b1d8f9055b32605591fe5c